ios/AirshipConfig.plist

```diff
@@ -4,13 +4,13 @@
 <dict>
 	<key>detectProvisioningMode</key>
 	<true/>
-	<key>developmentAppKey</key>
+	<key>appKey</key>
 	<string>uulSSfTDQJ2r0PMpjRrhmQ</string>
-	<key>developmentAppSecret</key>
+	<key>appSecret</key>
 	<string>D4Bhf0HrQEehrPua74Tyiw</string>
-	<key>productionAppKey</key>
+	<key>appKey</key>
 	<string>55vypj0ARc6cN09MX7ogtQ</string>
-	<key>productionAppSecret</key>
+	<key>appSecret</key>
 	<string>EsSaqbdLSvmyC6kSBFJCtQ</string>
 </dict>
```

Comment:
Just FYI: This file shouldn't have been changed. It introduced [this](https://github.com/Expensify/App/issues/13631) regression. According to [this](https://expensify.slack.com/archives/C01RJ1RCK8D/p1669227659448549?thread_ts=1669222166.493339&cid=C01RJ1RCK8D) Slack message from Airship team, it is fine to have `appKey`, 
 `appSecret` for Android and `developmentAppKey`, `developmentAppSecret`, `productionAppKey`, `productionAppSecret` for iOS. [Here](https://github.com/Expensify/App/pull/13674) is a PR to fix this.

Reply:
Ahhh, that's a bit annoying that they don't align their SDKs. Nice find though and thanks for raising it.